forgerock-authenticator/src/main/java/org/forgerock/android/auth/biometric/DeviceCredentialFragment.java

```diff
@@ -0,0 +1,85 @@
+/*
```

Comment:
same comment as above,  we can convert this whole Fragment class to regular class if we use registerForActivityResult. but its up to you if you wanna do it . i already have a story for this in the backlog. https://bugster.forgerock.org/jira/browse/SDKS-1827

Reply:
I would prefer leave this change to be done with the ticket in the backlog.